validator/src/main/jkube/deployment.yml

```diff
@@ -2,18 +2,26 @@ spec:
   template:
     spec:
       containers:
-      - env:
-          - name: JAVA_OPTIONS
-            value: " -Dgreeting=@greeting@ "
-      livenessProbe:
-        httpGet:
-          path: /
-          port: 8161
-          scheme: HTTP
-        initialDelaySeconds: 180
-      readinessProbe:
-        httpGet:
-          path: /
-          port: 8161
-          scheme: HTTP
-        initialDelaySeconds: 10
\ No newline at end of file
+        - env:
+            - name: JAVA_OPTIONS
+              value: " -Dgreeting=@greeting@ "
+          livenessProbe:
+            httpGet:
+              path: /actuator/health/liveness
+              port: 8081
+              scheme: HTTP
+            initialDelaySeconds: 180
+          readinessProbe:
+            httpGet:
+              path: /actuator/health/readiness
+              port: 8081
+              scheme: HTTP
+            initialDelaySeconds: 10
+          lifecycle:
```

Comment:
I do not understand if this is really needed for production purposes @mcarlett  wdyt

Reply:
if I understood correctly those scripts needed to wait for the process termination right?

Reply:
As @mcarlett already said, the need is to test those properties:

 `# increase graceful shutdown timeout to 60 seconds (default is 45)
camel.main.shutdown-timeout = 60

# monitoring the DefaultShutdownStrategy behaviour
logging.level.org.apache.camel.impl.engine.DefaultShutdownStrategy = DEBUG` 

Take care that the "_DEBUG_" log level too, is for test purposes. The lifecycle is designed to collect the Pod log before the SIGKILL event is fired by the kubelet

Reply:
the question is: do we need to provide those properties in a production environment, I mean those pod hook implementation change the application behavior? thanks